src/GraphQL.Tests/Serialization/NewtonsoftJsonTests.cs

```diff
@@ -0,0 +1,101 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Threading.Tasks;
+using GraphQL.NewtonsoftJson;
+using Newtonsoft.Json.Linq;
+using Shouldly;
+using Xunit;
+
+namespace GraphQL.Tests.Serialization
+{
+    public class NewtonsoftJsonTests
+    {
+        private readonly object _example = new
```

Comment:
Why object?

Reply:
And `dynamic` below on line 98.

Reply:
var doesn't work here.  I can copy the type def from the other class...

Reply:
done